fr/orm/validation.rst

```diff
@@ -52,21 +52,21 @@ requêtées, définissez l'option ``validate`` à false::
         ['validate' => false]
     );
 
-The same can be said about the ``patchEntity()`` method::
+La méthode ``patchEntity()`` fonctionne de façon identique::
 
     $article = $articles->patchEntity($article, $newData, [
         'validate' => false
     ]);
 
-Creating A Default Validation Set
-=================================
+Créer un Ensemble de Validation par Défaut
+==========================================
 
-Validation rules are defined in the Table classes for convenience. This helps
-find what data should be validated in correspondence to where it will be saved.
+Les règles de validation sont définies dans la classe Table par commodité.
+Cela aide à trouver les données qui doivent être validées en correspondance
+avec l'endroit où elles seront sauvegardées.
 
-
-To create a default validation object in your table, create the
-``validationDefault()`` function::
+Pour créer un objet validation dans votre table, créez la fonction
+``validationDefault()`` ::
```

Comment:
@cake17 I don't think it makes any difference and there's already a lot of cases like this in the docs